docs/pieces.md

```diff
@@ -327,13 +322,13 @@ These are the important sub-sections of a query builder:
 
 ## REST APIs for pieces
 
-REST APIs are automatically made available for piece types. For more information, see [REST APIs](rest-apis.md).
+REST APIs are automatically made available for piece-types. For more information, see [REST APIs](rest-apis.md).
 
 ## Pieces and widgets
 
-Of course, it's helpful to be able to display pieces anywhere on the site. Sometimes we don't want a browseable piece page at all, sometimes we want to "tease" that piece as the "featured product" on the home page, and so on.
+Of course, it's helpful to be able to display pieces anywhere in your project. Sometimes you won't want a browseable piece page at all, sometimes or you may want to "tease" that piece as the "featured product" on the home page, and so on.
```

Comment:
Grammar: "sometimes or you"

Reply:
"initialized with" empty methods? The wording is still confusing to me.